### PR TITLE
"Local DataBase Conncetion"

### PR DESCRIPTION
**Pull request by abdelilahAchir**

### CODE REVIEW

File: mrscake/src/constants/api_url_path.js [hunk 1]
```diff
@@ -1 +1 @@
-export const PRODUCTS_API_URL = 'https://localhost:5001/api/products';
\ No newline at end of file
+export const PRODUCTS_API_URL = 'https://localhost:44315/api/products';
```
Comment (Parul76) [inline mrscake/src/constants/api_url_path.js:1]: @abdelilahAchir Why you changed localhost number from 5001 to 44315

File: Mrs_Cake/_Mrs_Cake_MrsCakeDbContextDataSet.xsd [hunk 2]
```diff
@@ -0,0 +1,556 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (Parul76) [inline Mrs_Cake/_Mrs_Cake_MrsCakeDbContextDataSet.xsd:1]: you wrote this code or come from any data insertion file? Can you please little bit elaborate?

File: Mrs_Cake/_Mrs_Cake_MrsCakeDbContextDataSet.Designer.cs [hunk 3]
```diff
@@ -0,0 +1,4963 @@
+﻿//------------------------------------------------------------------------------
```
Comment (Parul76) [inline Mrs_Cake/_Mrs_Cake_MrsCakeDbContextDataSet.Designer.cs:1]: @Alina-dev-front  or @abdelilahAchir Do I need to review this code? As this code came from tool?

File: Mrs_Cake/Startup.cs [hunk 4]
```diff
@@ -10,7 +10,7 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
-using Mrs_Cake.Services;
+using Mrs_Cake.MrsCakeData;
```
Comment (Parul76) [inline Mrs_Cake/Startup.cs:13]: @abdelilahAchir Why you changed from services to data? It means services was taking inbuilt data or data will take from databse? Please elaborate

**Review by Parul76 — APPROVED** [review]

**Review by Alina-dev-front — APPROVED** [review]

**Pull request merged**